opossum-tool/OpossumUI
# feat: replace "all views" integration tests with E2E tests

### Summary of changes

- replace remaining "all views" int by E2E tests
- delete any reference to integration tests
- delete any unused test helpers

### Context and reason for change

Closes #2254.

### How can the changes be tested

Run locally and see CI/CD.
Note that there are no more integration tests left now. So `yarn test:integration` no longer exists.
